varia/sass/child-theme/_config-child-theme-deep.scss

```diff
@@ -390,6 +381,6 @@ $config-footer: (
 			"css-var": map-deep-get($config-global, "font", "family", "primary", "css-var"),
 		),
 		"size": map-deep-get($config-global, "font", "size", "sm"),
-		"line-height": map-deep-get($config-global, "font", "line-height", "sm"),
```

Comment:
`$config-global` doesn't have this value, so I've changed it to `body` instead. Is that ok in this instance?